### DISCUSSION TITLE
How to chapter skip on Nvidia Shield?

### DISCUSSION THREAD

**Discussion by HDGuy** [moneytoo/Player#384] [Q&A]
You’d think it would be obvious but I can’t figure it out. I’m using a Harmony Hub and have all the proper buttons programmed into the remote- my long press arrows are “skip forward” and “skip back”. It should work (does in Kodi) but it doesn’t, it just advances/rewinds a few secs like the directional buttons. 

The chapters are properly indicated on the progress bar.

Any ideas?

**Comment by moneytoo** [comment]
There's currently no support for navigating or skipping chapters.

**Reply by HDGuy** [reply]
> There's currently no support for navigating or skipping chapters.

Don't show them and tempt me then. 😛

One more thing to add to the list I guess.
